assembly/src/ast/parsers/constants.rs

```diff
@@ -0,0 +1,262 @@
+use super::{Felt, LocalConstMap, ParsingError, StarkField, String, ToString, Token, Vec};
+use core::fmt::Display;
+
+// CONSTANT VALUE EXPRESSIONS
+// ================================================================================================
+
+/// An operation used in constant expressions
+#[derive(Debug, PartialEq)]
+enum Operation {
+    Add,
+    Sub,
+    Mul,
+    Div,
+    IntDiv,
+    LPar,
+    RPar,
+    Value(Felt),
+}
+
+impl Operation {
+    /// Returns operator from [Operation] based on provided character
+    pub fn get_operator(char: &char) -> Self {
+        match char {
+            '+' => Self::Add,
+            '-' => Self::Sub,
+            '*' => Self::Mul,
+            '/' => Self::Div,
+            '(' => Self::LPar,
+            ')' => Self::RPar,
+            _ => unreachable!("Invalid operator character"),
+        }
+    }
+}
+
+impl Display for Operation {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        use Operation::*;
+
+        match self {
+            Add => write!(f, "+"),
+            Sub => write!(f, "-"),
+            Mul => write!(f, "*"),
+            Div => write!(f, "/"),
+            IntDiv => write!(f, "//"),
```

Comment:
I think these are reversed (`IntDiv` is integer division) - but I actually think that I like it better this way. That is: `/` is field division and `//` is integer division.

Reply:
Ah, indeed, somehow they were swapped in my mind. So, should I change them as you suggested it in issue discussion or leave it like this making `/` an integer division and `//` a filed division?

Reply:
No - I actually like the reverse approach: `/` for field division and `//` for integer division. So, this place should be fine, but there is another place in the code where it needs to be flipped.